autopilot/interface.go

```diff
@@ -210,9 +206,11 @@ func init() {
 	}
 
 	// Fill available node metrics.
-	for _, h := range availableNodeMetrics {
-		AvailableNodeMetrics[h.Name()] = h
+	centrality, err := NewBetweennessCentralityMetric(runtime.NumCPU())
+	if err != nil {
+		panic(err)
 	}
+	AvailableNodeMetrics[centrality.Name()] = centrality
```

Comment:
is this ever used? should the rpcserver share this singleton?

Reply:
yeah, removed until needed